formtranslate/templates/formtranslate/home.html

```diff
@@ -1,10 +1,14 @@
+{% load hq_shared_tags %}
+{% load i18n %}
 <!DOCTYPE html>
 <html>
 <head>
-    <link rel="stylesheet" href="{{ STATIC_URL }}formtranslate/css/blueprint/screen.css" type="text/css" media="screen, projection">
-    <link rel="stylesheet" href="{{ STATIC_URL }}formtranslate/css/blueprint/print.css" type="text/css" media="print">
-    <!--[if lt IE 8]><link rel="stylesheet" href="{{ STATIC_URL }}formtranslate/css/blueprint/ie.css" type="text/css" media="screen, projection"><![endif]-->
+    <link rel="stylesheet" href="{% static 'formtranslate/css/blueprint/screen.css' %}" type="text/css" media="screen, projection">
+    <link rel="stylesheet" href="{% static 'formtranslate/css/blueprint/print.css' %}" type="text/css" media="print">
+    <!--[if lt IE 8]><link rel="stylesheet" href="{% static 'formtranslate/css/blueprint/ie.css" type="text/css" media="screen, projection"><![endif]-->
```

Comment:
we could probably just drop this. anyone using less than IE 8 will have the page break for many other reasons